github3/pulls.py

```diff
@@ -313,14 +313,16 @@ def issue_comments(self, number=-1, etag=None):
         return self._iter(int(number), url, IssueComment, etag=etag)
 
     @requires_auth
-    def merge(self, commit_message='', sha=None):
+    def merge(self, commit_message='', squash=False, sha=None):
```

Comment:
Please move `squash` to the end. Since `sha` was there first, someone might be doing:

```py
pr.merge(my_message, sha)
```

So squash should come last otherwise we will severely break existing code.